core/util.go

```diff
@@ -100,3 +101,15 @@ func defToState(def *pb.Definition) (llb.State, error) {
 
 	return llb.NewState(defop), nil
 }
+
+func absPath(workDir string, path_ string) string {
```

Comment:
nit: is `path_` a typo?

Reply:
This was me getting frustrated with the `path` package name collision. 😂 

Refactored later in https://github.com/dagger/dagger/pull/3217/commits/b0b1f6b864f61378241d67eae25df5442b1458e0